populate_songs.py

```diff
@@ -41,6 +44,12 @@ def populate_songs(spreadsheetId, inEvents, shouldPrintTitle):
             songName = re.sub(TAGS_REGEX, '', inEvent.text)
             actualSongName = get_full_song_name(spreadsheetId, songName)
             if actualSongName is not None:
+                if not hasStarted:
+                    hasStarted = True
+                else:
+                    if shouldDelay:
```

Comment:
Actually, since we already know that the rate limit error will happen every 10 songs, would it be possible to instead do something like:

```python
numSongs = 0

if actualSongName is not None:
    if numSongs != 0 and numSongs % 10 == 0:
        time.sleep(7)
    numSongs += 1
```

Maybe if we do it this way we don't even need an explicit `--delay` flag

Reply:
Updated.